proto/buf.gen.pulsar.yaml

```diff
@@ -8,13 +8,14 @@ managed:
       - buf.build/cosmos/gogo-proto
       - buf.build/cosmos/cosmos-proto
     override:
+      buf.build/tendermint/tendermint: buf.build/gen/go/tendermint/tendermint/protocolbuffers/go
 plugins:
   - name: go-pulsar
     out: ../api
     opt: paths=source_relative
   - name: go-grpc
     out: ../api
     opt: paths=source_relative
-  - name: go-cosmos-orm
-    out: ../api
-    opt: paths=source_relative
+  # - name: go-cosmos-orm
+  #   out: ../api
+  #   opt: paths=source_relative
```

Comment:
I assume this is failing?

Reply:
It shouldn't be as it isn't used. I'll readd it.